test/FlexVotingClient.t.sol

```diff
@@ -29,7 +31,8 @@ contract FlexVotingClientTest is Test {
   uint256 MAX_VOTE_TYPE = uint256(type(GCF.VoteType).max);
 
   function setUp() public {
-    token = new GovToken();
+    if (_timestampClock()) token = new TimestampGovToken();
+    else token = new GovToken();
```

Comment:
Setup will deploy a timestamp token based on the value of a new virtual function. Since the token determines the governor's clock too, this is all that's needed to switch the whole governance system to a new clock implementation.